pigpio.js

```diff
@@ -292,6 +303,92 @@ class Notifier {
 
 module.exports.Notifier = Notifier;
 
+/* ------------------------------------------------------------------------ */
+/* Waves                                                                    */
+/* ------------------------------------------------------------------------ */
+
+Gpio.prototype.gpioPulse = pigpio.gpioPulse;
```

Comment:
What is the purpose of this line of code?

Reply:
Removed in abe1d13...Useless